src/app/components/driver-register/driver-register.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, OnInit } from '@angular/core';
+import { UserService } from 'src/app/services/user-service/user.service';
+import { BatchService } from 'src/app/services/batch-service/batch.service';
+import { Batch } from 'src/app/models/batch';
+import { Router } from '@angular/router';
+import { ValidationService } from 'src/app/services/validation-service/validation.service';
+import { User } from 'src/app/models/user';
+
+@Component({
+	selector: 'app-driver-register',
+	templateUrl: './driver-register.component.html',
+	styleUrls: ['./driver-register.component.css']
+})
+
+/**
+ * This is the Driver Register
+ */
+
+export class DriverRegisterComponent implements OnInit {
+
+	batches: Batch[] = [];
+	batch: Batch = new Batch();
+	user: User = new User();
```

Comment:
Both here and in the rider register, why do we need a batch independent of the user? Doesn't the user already have a batch field?

Reply:
-- an instance of a batch object is to be assigned in the user model --